index/scorch/scorch.go

```diff
@@ -191,6 +192,23 @@ func (s *Scorch) openBolt() error {
 		}
 	}
 
+	s.numSnapshotsToKeep = NumSnapshotsToKeep
```

Comment:
hmmn, i don't think round tripping the value through json is the right approach either